otherlibs/runtime_events/runtime_events.mli

```diff
@@ -133,6 +133,60 @@ module Timestamp : sig
     val to_int64 : t -> int64
 end
 
+module Type : sig
+  type 'a t
+  (** The type for an user event content type *)
+
+  val event : unit t
+  (** An event has no data associated with it *)
```

Comment:
nit: Calling this an "event" doesn't seem appropriate. `span` and `counter` are also "events". Can we pick a different name for this event? Given that this is a nullary event can we call this `unit`?

Reply:
Looks good!